source/query/atlas-search.txt

```diff
@@ -12,11 +12,23 @@ Run Atlas Search Queries
    :depth: 1
    :class: singlecol
 
-You can run Atlas Search :pipeline:`$search` and :pipeline:`$searchMeta`
-queries from |compass| if you :ref:`created the Atlas Search index
-<compass-create-search-index>` for data on the Atlas cluster. You can
-also run :pipeline:`$vectorSearch` queries from |compass| if you have an
-existing Atlas Vector search index on your data in Atlas. 
+Atlas Search queries take the form of an :ref:`aggregation pipeline stage 
```

Comment:
(minor)
"are run as" rather than "take the form of" 
Optional thought.

Reply:
that sounds better. updated!